### PR TITLE
FEAT: Create A Base Template of The Interface With Json Data

### PR DESCRIPTION
**Pull request by Reddy017**
### Description of Changes:

Create a base template of the interface with json data

### Type of Changes:

- [ ] Documentation (Add Documentation to Existing Code)
- [ ] Bug Fix (Non-Breaking Changes Which Fixes an Issue)
- [x] New Feature (Non-Breaking Changes Which Adds Functionality)
- [ ] Improvement (Non-Breaking Changes Which Improves the Structures, Performance, etc...)
- [ ] **Breaking Change** (Breaking Changes that Would Cause Existing Functionality to NOT Work as Expected)

### To-Do or Other Concerns:

- (NONE)

### CODE REVIEW

**Review by XiaoyuYang18 — APPROVED** [review]

File: EasyContact.app/client/src/index.js [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
-import reportWebVitals from './reportWebVitals';
+import React from "react";
+import ReactDOM from "react-dom";
+import bootstrap from "bootstrap"; // eslint-disable-line no-unused-vars
+import Inbox from "./components/Inbox";
+import HeaderHtml from "./components/templates/HeaderHtml";
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
-  <React.StrictMode>
```
Comment (xiaoshaoyc) [inline EasyContact.app/client/src/index.js:9]: Maybe keep `React.strictMode` is a better choice?

**Review by xiaoshaoyc — COMMENTED** [review]
Clear structure and good naming!

We can discuss some details in the next meeting.

**Review by wangz44 — APPROVED** [review]
nice

**Review by xiaokeai55 — APPROVED** [review]

**Review by xiaoshaoyc — APPROVED** [review]

**Review by ZHIHANCHEN03 — APPROVED** [review]
Great template, just need to add useable functions to ensure that those buttons are functional

**Pull request merged**